mewarpx/tests/test_custom_criteria.py

```diff
@@ -0,0 +1,110 @@
+"""Test the custom criteria using capacitive discharge"""
```

Comment:
Note that `test_custom_criteria.py` was renamed to `test_mepicmi.py` in a different PR, so this is adding that file back to the tests. I don't think we need to do that, right?